project-codeflare/codeflare-common
# Support package refactor

# Issue link
https://github.com/project-codeflare/codeflare-operator/issues/237

# What changes have been made
- Moved the support files from the Codeflare Operator Repo to this new separate repo for common functions.
- Added Makefile and moved `verify-imports.sh` in order to be able to verify imports are correct.

Questions:
- Do we want to group the files further within the support directory?
- We should have an OWNERS file - Who should be added as approvers and reviewers?
- Do we want to add PR checks for this repo now?

# Verification Steps
- Once the [Codeflare Operator repo](https://github.com/project-codeflare/codeflare-operator/pull/336) is updated to reflect these changes we can verify that everything works as expected.

## Checks
- Testing Strategy
